schemas/context/phonenumber.schema.json

```diff
@@ -73,5 +73,5 @@
       "$ref": "https://ns.adobe.com/xdm/common/auditable#/definitions/auditlog"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎
Validity doesn't document what it is consistent WITH, so it's useful
Also need to remove reference to auditable